hadoop-hdds/docs/content/beyond/Containers.md

```diff
@@ -111,23 +117,32 @@ OZONE-SITE.XML_ozone.enabled=True
 #...
 ```
 
-As you can see we use naming convention. Based on the name of the environment variable, the appropariate hadoop config XML (`ozone-site.xml` in our case) will be generated by a [script](https://github.com/apache/hadoop/tree/docker-hadoop-runner-latest/scripts) which is included in the `hadoop-runner` base image.
+As you can see we use naming convention. Based on the name of the environment variable, the
+appropriate hadoop config XML (`ozone-site.xml` in our case) will be generated by a
+[script](https://github.com/apache/hadoop/tree/docker-hadoop-runner-latest/scripts) which is
+included in the `hadoop-runner` base image.
 
-The [entrypoint](https://github.com/apache/hadoop/blob/docker-hadoop-runner-latest/scripts/starter.sh) of the `hadoop-runner` image contains a helper shell script which triggers this transformation and cab do additional actions (eg. initialize scm/om storage, download required keytabs, etc.) based on environment variables.
+The [entrypoint](https://github.com/apache/hadoop/blob/docker-hadoop-runner-latest/scripts/starter.sh)
+of the `hadoop-runner` image contains a helper shell script which triggers this transformation and
+can do additional actions (eg. initialize scm/om storage, download required keytabs, etc.)
+based on environment variables.
 
 ## Test/Staging
 
-The `docker-compose` based approach is recommended only for local test not for multi node cluster. To use containers on a multi-node cluster we need a Container Orchestrator like Kubernetes.
+The `docker-compose` based approach is recommended only for local test, not for multi node cluster.
+To use containers on a multi-node cluster we need a Container Orchestrator like Kubernetes.
 
 Kubernetes example files are included in the `kubernetes` folder.
```

Comment:
Should we be more specific like hadoop-ozone/dist/target/ozone-*/kubernetes

Reply:
The documentation is included in the tar file where there is no target/dist. I think it's fine to use path relative to the distribution tar. We can be more specific with the `kubernetes/examples` directory name..